cmake/external/abseil-cpp/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ include(FetchContent)
 set(ABSL_PROPAGATE_CXX_STD 1)
 set(BUILD_TESTING 0)
 
+if(MSVC)
+add_compile_options(/WX-)
```

Comment:
But who set "/WX"?  By default warnings are not errors. I only set "/WX" after all third-party libraries were built. i.e. after this file is included.

Reply:
See https://github.com/microsoft/onnxruntime/blob/master/cmake/CMakeLists.txt#L1163 

"WX" is set at line 1163. But abseil is at line 111. Much before it.

Reply:
"/WX" is set by the internal windowsAI build

Reply:
Would it be better to use "target_compile_options" and specify a target?

Reply:
@ryanlai2 absl has tens of targets.